package.json

```diff
@@ -13,7 +13,8 @@
   "license": "ISC",
   "dependencies": {
     "@noir-lang/backend_barretenberg": "^0.21.0",
-    "@noir-lang/noir_js": "^0.21.0"
+    "@noir-lang/noir_js": "^0.21.0",
+    "alchemy-sdk": "^3.1.0"
```

Comment:
Move to devDependencies (also remove for current PR)

Reply:
Remove alchemy for now

Reply:
removed